src/extension/alterschema/linter/non_applicable_enum_validation_keywords.h

```diff
@@ -0,0 +1,83 @@
+class NonApplicableEnumValidationKeywords final : public SchemaTransformRule {
+public:
+  NonApplicableEnumValidationKeywords()
+      : SchemaTransformRule{
+            "enum_validation_keywords_default",
+            "Setting validation keywords that do not apply to any item in "
+            "`enum` is considered an "
+            "anti-pattern, as the enumeration choices already imply their "
```

Comment:
```suggestion
            "anti-pattern"
```

I think we can leave the last part off. I think its clear enough without it